extensions/amp-animation/0.1/test/test-web-animations.js

```diff
@@ -292,6 +292,65 @@ describes.sandboxed('MeasureScanner', {}, () => {
       {opacity: '1'},
     ]);
   });
+
+
+  describes.fakeWin('createRunner', {amp: 1}, env => {
+    let resources;
+    let amp1, amp2;
+
+    beforeEach(() => {
+      resources = env.win.services.resources.obj;
+      amp1 = env.createAmpElement();
+      amp2 = env.createAmpElement();
+      resources.add(amp1);
+      resources.add(amp2);
+    });
+
+    function createRunner(spec) {
+      const targets = {target1, target2, amp1, amp2};
+      const scanner = new MeasureScanner(window, {
+        resolveTarget: name => targets[name] || null,
+      }, true);
+      scanner.scan(spec);
+      return scanner.createRunner(resources);
+    }
+
+    it('should unblock non-AMP elements immediately', () => {
+      let runner;
+      const runnerPromise = createRunner([
+        {target: target1, keyframes: {}},
+        {target: target2, keyframes: {}},
+      ]).then(res => {
+        runner = res;
+      });
+      return Promise.race([runnerPromise, Promise.resolve()]).then(() => {
```

Comment:
No technical reason. I just thought it communicates what I was waiting for better - i.e. `runnerPromise` is not resolved in that case.